crates/bevy_light/src/directional_light.rs

```diff
@@ -240,3 +240,55 @@ pub fn update_directional_light_frusta(
             .collect();
     }
 }
+
+/// This component marks a [`DirectionalLight`] entity for changing the size and intensity of the sun disk.
```

Comment:
This sentence is a little weird sounding to me, but I don't know what would be better.